### ISSUE TITLE
weather no longer working since last week(mid march)

### ISSUE THREAD

**Issue by ale82to** [quodroc/HumbleNewTabPage#43]
weather is no longer working i ve got error loading weather :(

**Comment by quodroc** [comment]
Nothing I can do at the moment but I'm planning on adding OpenWeatherMap as an option instead of Yahoo Weather. If it is consistently failing with you particular location you may want to try a nearby city instead.

**Comment by smbkr** [comment]
This is failing for me now, with location set to London.

**Comment by AdnanHodzic** [comment]
With this bug being open for almost 2 years I was wondering are there any updates?

Needless to say, I have same issue with location set to Amsterdam. Issue is present on both Linux (Debian Buster) and MacOS (Mojave).

**Comment by atwright147** [comment]
I don't think this bug is relevant now, the new weather breakage is permanent and relates to a new issue.

@quodroc I suggest closing this.
